frontend/src/views/Timeline.vue

```diff
@@ -6,7 +6,7 @@
 </template>
 
 <script>
-import TimelineCard from "../components/TimelineCard.vue";
+import TimelineCard from "../components/Timelinecard.vue";
```

Comment:
Undo this change

Reply:
The component name is Timelinecard.
It was written wrongly in Sandesh's PR. So i changed it.

Reply:
It shouldn't be. Change that also

Reply:
done